src/models/Player.js

```diff
@@ -6,6 +6,14 @@ const { isNullUUID, fixUpUUID, stripUUID } = require('./util');
 const { Op } = Sequelize;
 
 class Player {
+  /**
+ * Returns a list of results from a players query in result and the number of total matches in totalCount.
+ * @param {number} limit - Max number of results to return.
+ * @param {number} offset - Number to offset the first index of results by, useful for pagination.
+ * @param {string} order - ASC or DESC, determines the order of players returned in result ordered by first joined date.
+ * @param {boolean} matchAll - If true; searchPlayerName's elements all should match lastSeenName to be returned. If false, any can match.
+ * @param {string[]} searchPlayerName - List of strings to use for filtering by lastSeenName 
```

Comment:
nit: might be good to mention that these args are optional. you can do that in jsdoc like so:

```
@param {boolean} [matchAll=false] - foo bar